### ISSUE TITLE
jsx-jcs-no-undef not catching missing components

### ISSUE THREAD

**Issue by igl** [vkbansal/eslint-plugin-jsx-control-statements#17]
`jsx-control-statements/jsx-jcs-no-undef` is not catching missing Components in JSX. Other vars seem to work. I am using vscode (dbaeumer.vscode-eslint v1.2.11) I am not sure when this broke since i don't remember updating lint nor this plugin lately.

![screen shot 2017-05-19 at 19 31 55](https://cloud.githubusercontent.com/assets/293519/26259566/e268c8d4-3cc9-11e7-969a-0d508683e453.png)

### Config:
```yml
extends: "eslint:recommended"
parser: "babel-eslint"
plugins: [ "react", "flowtype", "jsx-control-statements" ]
env:
    es6: true
    node: true
    browser: true
rules:
    ...
    no-undef: 0
    react/jsx-no-undef: 0
    jsx-control-statements/jsx-jcs-no-undef: 2
    ...
```

### Versions:
```sh
$ yarn list | grep eslint
├─ babel-eslint@7.2.3
├─ eslint-plugin-flowtype@2.33.0
├─ eslint-plugin-jsx-control-statements@2.1.1
├─ eslint-plugin-react@7.0.1
├─ eslint@3.19.0
```

**Comment by libasoles** [comment]
This worked for me:
`"react/jsx-no-undef": [true, { "allowGlobals": true }]`

Ref: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-undef.md
```
"react/jsx-no-undef": [<enabled>, { "allowGlobals": <boolean> }]
```
